bundle/edu.gemini.spdb.rollover.servlet/src/main/scala/edu/gemini/rollover/servlet/RolloverServlet.scala

```diff
@@ -55,10 +55,12 @@ final class RolloverServlet(site: Site, odb: IDBDatabaseService, user: java.util
     <obs>
       <id>{ ro.id.toString }</id>
       <partner>{ ro.partner }</partner>
-      <target>
-        <ra>{ ro.target.coords.map(_.ra).getOrElse(0.0) }</ra>
-        <dec>{ ro.target.coords.map(_.dec).getOrElse(0.0) }</dec>
-      </target>
+      {ro.targets.map { t =>
+        <target>
+          <ra>{  t.coords.map(_.ra).getOrElse(0.0) }</ra>
+          <dec>{ t.coords.map(_.dec).getOrElse(0.0) }</dec>
+        </target>
+      }}
```

Comment:
For rollover, probably just the base position is what we want.  It's about reserving time for the observation during the semester so we don't schedule too many new observations for the corresponding part of the sky.  This may just work as is, but we'd need to check the consumer of this to make sure it can handle multiple targets if we leave it this way.  I guess it *might* just read the first target which would be fine.